CMakeLists.txt

```diff
@@ -7,7 +7,11 @@ add_subdirectory(lib/open-simulation-interface)
 get_directory_property(OSI_VERSION_MAJOR DIRECTORY lib/open-simulation-interface DEFINITION VERSION_MAJOR)
 get_directory_property(OSI_VERSION_MINOR DIRECTORY lib/open-simulation-interface DEFINITION VERSION_MINOR)
 get_directory_property(OSI_VERSION_PATCH DIRECTORY lib/open-simulation-interface DEFINITION VERSION_PATCH)
-set(OSIVERSION "${OSI_VERSION_MAJOR}.${OSI_VERSION_MINOR}.${OSI_VERSION_PATCH}")
+# set(OSIVERSION "${OSI_VERSION_MAJOR}.${OSI_VERSION_MINOR}.${OSI_VERSION_PATCH}")
+# CMOSI bugfix, CMOSI interface expects OSI 3.0.0, although supporting 3.5.0
+set(OSIVERSION "3.0.0")
+set(OSMPVERSION "1.0.0" CACHE STRING "OSMP Version String")
```

Comment:
The current OSMP version is 1.4.0. It should be fully supported as far as I know. Does CarMaker require it to be 1.0.0?

Reply:
Yes exactly, this is why I had to fake in 1.0.0, same as for OSI - Only for testing purposes.